test/add.js

```diff
@@ -12,4 +13,18 @@ describe('add', function() {
     eq(incr(42), 43);
   });
 
+  describe('properties', function () {
+    jsc.property("commutative", "integer", "integer", function (a, b) {
+      return R.add(a,b) === R.add(b,a);
+    });
+
+    jsc.property("associative", "integer", "integer", "integer", function (a, b, c) {
+      return R.add(a, R.add(b, c)) === R.add(R.add(a, b), c);
+    });
+
+    jsc.property("neutral element", "integer" , function (a) {
+      return R.add(a, 0) === a;
+    });
```

Comment:
Probably "identity element" instead.

But also this should either be described here as the "right identity element" with a comparable "left identity element", or they should be combined with something like

```js
  R.add(a, 0) === a && R.add(0, a) === 0;
```